spec/support/resource_list_to_jstonapi.rb

```diff
@@ -16,6 +16,7 @@ def resource_list_to_jsonapi(resource_list, **opts)
     },
     include: opts[:include],
     meta: opts[:meta],
+    links: opts[:links],
```

Comment:
I know we're just adding one line here, but surely we should follow the boy-scout principle and fix the name of this file, no? ;)

Reply:
Didn't even notice